mkdocs.yml

```diff
@@ -12,10 +12,12 @@ nav:
     - Generate a Random BED File: generate_random.md
     - Bedshift All BED Files in Folder: bedshift_all.md
     - Evaluate Similarity Scores: evaluation_guide.md
+    - \"From File\ Features": from_file.md
```

Comment:
Why the escapes here?

Reply:
I was attempting to escape the quotation marks in case YAML didn't like them, but I misplaced the second quotation. Are the escapes necessary?

Reply:
Just say *From-file features* with no quotes